sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlan.scala

```diff
@@ -231,6 +269,11 @@ abstract class SparkPlan extends QueryPlan[SparkPlan] with Logging with Serializ
   }
 }
 
+object SparkPlan {
+  private[execution] val subqueryExecutionContext = ExecutionContext.fromExecutorService(
```

Comment:
`BroadcastHashJoin` defines a ThreadPool for broadcasting. I am moving that as part of #11083 into `exchange.scala`. We could use that one.